src/Surfnet/AzureMfa/Application/Service/AzureMfaService.php

```diff
@@ -199,6 +203,20 @@ public function handleResponse(Request $request): User
 
         $attributes = $assertion->getAttributes();
 
+        // TODO: This should be a config parameter in institutions.yaml instead of string matching
+        // If the IDP was an AzureAD endpoint (the entityID or Issuer starts with https://login.microsoftonline.com/, or preferably an config parameter in institutions.yaml)
+        // the SAML response attribute 'http://schemas.microsoft.com/claims/authnmethodsreferences'
+        // should contain 'http://schemas.microsoft.com/claims/multipleauthn'
+        if (strpos($azureMfaIdentityProvider->getSsoUrl(), AZUREAD_LOGIN_URL) === 0) {
```

Comment:
The constant `AZUREAD_LOGIN_URL` isn't going to work. You could change it to `self::AZUREAD_LOGIN_URL` to prevent exceptions. This new situation isn't reflected in the tests yet which still needs to be done in order to let the build pass.

Reply:
I've updated the constant. Can you add this to the tests?